analysis/topeft_run2/make_cr_and_sr_plots.py

```diff
@@ -859,6 +958,10 @@ def make_all_sr_data_mc_plots(dict_of_hists,year,save_dir_path):
             samples_to_rm_from_mc_hist.append(sample_name)
         if sample_name not in data_sample_lst:
             samples_to_rm_from_data_hist.append(sample_name)
+        #if "nonprompt" not in sample_name:
```

Comment:
Remove if not needed.

Reply:
The reason I left it as comment is that was my original logic in `ttA` analysis where I was only using `sumw2` hist for `nonprompt` pieces to calculate the stat error. I left it in case people want to go back to this logic in the future again. What do you think?

Reply:
I have done this now.

Reply:
@anpicci any final comments on this?